stable/mongodb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mongodb
-version: 5.1.3
+version: 5.1.4
```

Comment:
As you are including a new feature, you should update the minor version.

```suggestion
version: 5.2.0
```

Reply:
done!